src/nova_session.erl

```diff
@@ -86,7 +86,7 @@ delete(Req, Key) ->
     case get_session_id(Req) of
         {ok, SessionId} ->
             Mod = get_session_module(),
-            Mod:delete_value(SessionId, Key),
+            Mod:delete(SessionId, Key),
```

Comment:
This should be delete_value. It uses the callback-functions for operations like this.

Reply:
This goes for all changes in this file.